modules/linear_boltzmann_solvers/discrete_ordinates_curvilinear_solver/lbs_curvilinear_solver.cc

```diff
@@ -228,7 +228,7 @@ DiscreteOrdinatesCurvilinearSolver::PerformInputChecks()
                 log.LogAllError() << "D_DO_RZ_SteadyState::SteadyStateSolver::"
                                      "PerformInputChecks : "
                                   << "mesh contains boundary faces with outward-oriented unit "
-                                  << "normal vector " << (-1 * unit_normal_vectors[d]).PrintS()
+                                  << "normal vector " << (-1 * unit_normal_vectors[d]).PrintStr()
                                   << " with vertices characterised by v(" << d << ") != 0.";
                 Exit(EXIT_FAILURE);
```

Comment:
Another spot we should `throw` instead of `Exit`.

Reply:
so do you want me to use:
`throw std::logic_error(msg)
`?

Reply:
or `std::runtime_error(msg)`?

Reply:
i used logic error